guestbook/README.md

```diff
@@ -4,704 +4,294 @@
+### Creating the Redis Slave Deployment
+Deployments scale based off of the configurations set in the manifest file. In this case, the Deployment object specifies two replicas. 
 
-This example has been configured to use the DNS service by default.
+If there are not any replicas running, this Deployment would start the two replicas on your container cluster. Conversely, if there are more than two replicas are running, it would scale down until two replicas are running. 
 
-If your cluster does not have the DNS service enabled, then you can use environment variables by setting the
-`GET_HOSTS_FROM` env value in both
-[redis-slave-deployment.yaml](https://git.k8s.io/examples/guestbook/redis-slave-deployment.yaml) and [frontend-deployment.yaml](https://git.k8s.io/examples/guestbook/frontend-deployment.yaml)
-from `dns` to `env` before you start up the app.
-(However, this is unlikely to be necessary. You can check for the DNS service in the list of the cluster's services by
-running `kubectl --namespace=kube-system get rc -l k8s-app=kube-dns`.)
-Note that switching to env causes creation-order dependencies, since Services need to be created before their clients that require env vars.
+1. Apply the Redis Slave Deployment from the `redis-slave-deployment.yaml` file:
 
-#### Create a Deployment
+        kubectl apply -f redis-slave-deployment.yaml
 
-Second, create the redis master pod in your Kubernetes cluster by running:
+include code.html language="yaml" file="redis-slave-deployment.yaml" ghlink="/docs/tutorials/docs/tutorials/stateless-application/redis-slave-deployment.yaml"
 
-```console
-$ kubectl create -f guestbook/redis-master-deployment.yaml
-deployment "redis-master" created
-```
+{:start="2"}
+2. Query the list of Pods to verify that the Redis Slave Pods are running:
 
-You can see the Deployment for your cluster by running:
+        kubectl get pods
 
-```console
-$ kubectl get deployments
-NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-redis-master   1         1         1            1           27s
-```
+   The response should be similar to this:
 
-Then, you can list the pods in the cluster, to verify that the master is running:
+        NAME                            READY     STATUS              RESTARTS   AGE
+        redis-master-1068406935-3lswp   1/1       Running             0          1m
+        redis-slave-2005841000-fpvqc    0/1       ContainerCreating   0          6s
+        redis-slave-2005841000-phfv9    0/1       ContainerCreating   0          6s
+        
+### Creating the Redis Slave Service
+The guestbook application needs to communicate to Redis workers to read data. To make the Redis workers discoverable, you need to set up a Service. A Service provides transparent load balancing to a set of Pods.
```

Comment:
slave vs. workers: You need to unify the terminology. It's ok if you decide to change slave work (incl. filenames) to worker.

I already use the word `worker` and not slave (that only comes from imported files/filenames) in GKE guestbook: https://cloud.google.com/container-engine/docs/tutorials/guestbook